digitalocean/baseapi.py

```diff
@@ -97,6 +97,32 @@ def __perform_request(self, url, type=GET, params=None):
 
         return requests_method(url, **kwargs)
 
+    def __deal_with_pagination(self, url, method, params, data):
+        """
+            Perform multiple calls in order to have a full list of elements
+            when the API are "paginated". (content list is divided in more
+            than one page)
+        """
```

Comment:
Correct, the lack of inconsistency is the confusing part for me. In other place it was changed to single quotes. Don't worry, it is not a big issue as long as we can change it 👍 I suggest to just commit the real changes needed. I would prefer double quote btw.